core/lib/libtomcrypt/src/ciphers/sub.mk

```diff
@@ -1,5 +1,13 @@
 cflags-y += -Wno-unused-parameter
 
+ifeq ($(CFG_CRYPTO_AES_ARM64_CE),y)
+srcs-y += aes_arm64_ce.c
+cflags-aes_arm64_ce.c-y += -march=armv8-a+crypto
```

Comment:
On 14 June 2015 at 07:36, Victor Chong <notifications@github.com> wrote:

> In core/lib/libtomcrypt/src/ciphers/sub.mk
> <https://github.com/OP-TEE/optee_os/pull/341#discussion_r32376595>:
>
> > @@ -1,5 +1,13 @@
> >  cflags-y += -Wno-unused-parameter
> >
> > +ifeq ($(CFG_CRYPTO_AES_ARM64_CE),y)
> > +srcs-y += aes_arm64_ce.c
> > +cflags-aes_arm64_ce.c-y += -march=armv8-a+crypto
>
> -mgeneral-regs-only flag only covers platform-specific code? Will it
> include aes_arm64_ce.c?
>
>
​It should cover *all* code since otherwise, GCC may decide to spill data
to FP/SIMD​ registers at any point in the program.